README.md

```diff
@@ -1,2 +1,88 @@
 # app.js
-GitHub Apps toolset for Node.js
+
+> Official GitHub Platform for GitHub Apps in Node.js
+
+## GitHub Apps
+
+`@octokit/app.js` allows to authenticate as GitHub Apps, installations, listen
+to webhooks, and respond to them using the GitHub API.
+
+### Authenticating as an App
+
+In order to authenticate as a GitHub App, you need to generate a Private Key
+and use it to sign a JSON Web Token (jwt) and encode it. 
+
+```js
+const App = require('@octokit/app.js')
+
+const app = new App({id, privateKey})
+const appAuth = await app.appAuth()
+
+// Example of using authetincated app to GET an individual installation
+// https://developer.github.com/v3/apps/#find-installations
+const installation = await appAuth.request({
+  method: 'GET',
+  url: '/repos/:owner/:repo/installation',
+  headers: { accept: 'application/vnd.github.machine-man-preview+json' },
+  owner: 'hiimbex',
+  repo: 'testing-things'
+})
+
+// contains the installation id necessary to authenticate as an installation
+const installationId = installation.data.id
+```
+
+### Authenticating as an Installation
+
+Once you have authenticated as a GitHub App, `auth` here, you can use that
+in order to request an installation access token. This token is scoped for
+your specific app and expires after an hour.
+
+```js
+const app = require('@octokit/app.js')
+
+const app = new App({id, privateKey})
+const installation = await app.requestToken(app.appAuth(), {installationId: 123})
```

Comment:
why not just `await app.requestToken({installationId: 123})`? The method could get a JWT internally, the `app` instance variable has the app id and the private key as internal state